### ISSUE TITLE
Clarify twisted boot logs

### ISSUE THREAD

**Issue by maaktweluit** [golemfactory/clay#2365]
Please check if we can make this block of logs only debug, twisted should have enough on 1 or 2 info's to inform ELI5 what it is doing.

```2018-03-06 15:30:20 INFO     twisted                             No extra node router roles 
2018-03-06 15:30:20 INFO     twisted                             RouterServiceSession ready [no on_ready configured] 
2018-03-06 15:30:20 INFO     twisted                             Registered 20 procedures 
2018-03-06 15:30:20 INFO     twisted                             Using node shutdown triggers ['shutdown_on_shutdown_requested'] from configuration 
2018-03-06 15:30:20 INFO     twisted                             Configuring node from local configuration ... 
2018-03-06 15:30:20 INFO     twisted                             Starting 1 workers ... 
2018-03-06 15:30:20 INFO     twisted                             Router worker "worker-001" starting .. 
2018-03-06 15:30:21 INFO     twisted                             Started Router worker "worker-001" on node "None" [crossbar.worker.router.RouterWorkerSession / CPython-KQueueReactor] 
2018-03-06 15:30:21 INFO     twisted                             Router worker "worker-001" session 1532405869332247 initializing .. 
2018-03-06 15:30:21 INFO     twisted                             Registered 39 procedures 
2018-03-06 15:30:21 INFO     twisted                             Router worker "worker-001" session ready 
2018-03-06 15:30:21 INFO     twisted                             Router worker "worker-001" process 28164 started 
2018-03-06 15:30:21 INFO     twisted                             RouterServiceSession ready [configured on_ready fired] 
2018-03-06 15:30:21 INFO     twisted                             Realm 'golem' started 
2018-03-06 15:30:21 INFO     twisted                             Router "worker-001": realm 'realm-001' (named 'golem') started 
2018-03-06 15:30:21 INFO     twisted                             role role-001 on realm realm-001 started 
2018-03-06 15:30:21 INFO     twisted                             Router "worker-001": role 'role-001' (named 'anonymous') started on realm 'realm-001' 
2018-03-06 15:30:21 INFO     twisted                             WampWebSocketServerFactory starting on 61000 
2018-03-06 15:30:21 INFO     twisted                             Router "worker-001": transport 'transport-001' started 
2018-03-06 15:30:21 INFO     twisted                             Local node configuration applied successfully! 
2018-03-06 15:30:21 INFO     twisted                             Starting factory <autobahn.twisted.websocket.WampWebSocketClientFactory object at 0x110d7edd8> 
2018-03-06 15:30:21 INFO     twisted                             session "8307743868352033" joined realm "golem" 
```

**Comment by Elfoniok** [comment]
Original problem was solved some time ago, but problem still occur when one use debug switch for golem. Since this logs are third party libraries logs, we are not interested very much in them, however if they are need ed in rare cases it should be possible to show them, hence this commit:
https://github.com/golemfactory/golem/pull/2810
